mappings/net/minecraft/client/gui/screen/ingame/BrewingScreen.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_790 net/minecraft/client/gui/screen/ingame/BrewingStandScreen
```

Comment:
bad. old name is more yarn here

Reply:
![image](https://user-images.githubusercontent.com/62552372/96240607-bc29f000-0fbe-11eb-8ba7-b3062fb084da.png)